ai/src/tests/perf_data.py

```diff
@@ -0,0 +1,57 @@
+from io import StringIO
+import time
+
+import pandas as pd
+
+from proto.aiengine.v1 import aiengine_pb2
+import main
+from tests import common
+
+
+class PerfData():
+    def __init__(self):
+        print("Starting performance test for data\n")
+        self.aiengine = main.AIEngine()
+
+        self.trader_init_req = common.get_init_from_json(
+            init_data_path="../../test/assets/aiengine/api/trader_init.json",
+            pod_name="trader",
+        )
+        with open("../../test/assets/data/csv/trader.csv", "r", encoding="utf8") as trader_data:
```

Comment:
Can we use a bigger data set?

Reply:
There is a `COINBASE[...]` csv with 80KiB of data but to initialize an aiengine I would need to create a new file under `test/assets/aiengine/api/`. I went for the 1.8KiB trader data as it was the fastest and it is the data used for all the ai tests.

Reply:
New commit add coinbase data for performance check